### ISSUE TITLE
Save as waiting for GTEs

### ISSUE THREAD

**Issue by trymebytes** [GlotPress/GlotPress-WP#1869]
## What problem does this address?
A GTE cannot submit strings in waiting status, their translations get approved automatically. Some GTEs want to be able to mark some of their translations as waiting so they can get other GTEs to take a look before being approved.

## What is your proposed solution?
I'm proposing an additional button to allow GTEs save translations as waiting.

## Mockups/screenshots

<img width="1566" alt="Screenshot 2024-10-17 at 10 53 09" src="https://github.com/user-attachments/assets/46c9660a-a2ee-4908-b815-ccdf17d86b3e">

**Comment by amieiro** [comment]
[Discussion](https://wordpress.slack.com/archives/C02RP50LK/p1729129356442429) at "Making WordPress" Slack.

**Comment by Evirtec** [comment]
The **"Save as Waiting"** option would be very useful.

It allows GTEs to ensure the quality of their own translations. In small translation teams (with 1 or 2 GTEs like in Argentina), GTEs have to do both translation and review _simultaneously_.

That can lead to:
a. Typos
b. Uncertainty about certain translations

Having the ability to mark our own translated strings as **"Save as Waiting"** would let a colleague review them or allow us to review them later with a fresh perspective.

Additionally, while translating a series of strings, doubts might arise about the most accurate way to translate certain strings. Instead of skipping them, we could make an approximate translation and leave it in **"Save as Waiting"** to later seek information and review the strings in **"Waiting"**.

Until now, the only way to do this has been to approve the translation and then mark it as **"fuzzy"** which doubles the work for the platform.

**Comment by Bizware** [comment]
The **"Save as Waiting"** option would be very useful.

It allows GTEs to ensure the quality of their own translations. In small translation teams (with 1 or 2 GTEs like in Argentina), GTEs have to do both translation and review _simultaneously_.

That can lead to:
a. Typos
b. Uncertainty about certain translations

Having the ability to mark our own translated strings as **"Save as Waiting"** would let a colleague review them or allow us to review them later with a fresh perspective.

Additionally, while translating a series of strings, doubts might arise about the most accurate way to translate certain strings. Instead of skipping them, we could make an approximate translation and leave it in **"Save as Waiting"** to later seek information and review the strings in **"Waiting"**.

Until now, the only way to do this has been to approve the translation and then mark it as **"fuzzy"** which doubles the work for the platform.

**Comment by pedro-mendonca** [comment]
Features like these are great, but shouldn't be implemented with only w.org in mind.
GTE is a role on translate.w.org. GlotPress is beyond that, is used by many software companies.
The feature should be created with GlotPress core "Validator" permission in mind :-)

**Comment by amieiro** [comment]
> Features like these are great, but shouldn't be implemented with only w.org in mind. GTE is a role on translate.w.org. GlotPress is beyond that, is used by many software companies. The feature should be created with GlotPress core "Validator" permission in mind :-)

Yes, of course. In fact, this is the reason Tosin opened the issue here and not in the [Trac](https://meta.trac.wordpress.org/query?status=accepted&status=assigned&status=new&status=reopened&status=reviewing&component=Translate+Site+%26+Plugins&order=priority).
